modules/cluster-logging-must-gather-collecting.adoc

```diff
@@ -16,18 +12,19 @@ To collect {logging} information with `must-gather`:
 
 . Run the `oc adm must-gather` command against the {logging} image:
 +
-ifndef::openshift-origin[]
+If you are using OKD:
++
 [source,terminal]
 ----
-$ oc adm must-gather --image=$(oc -n openshift-logging get deployment.apps/cluster-logging-operator -o jsonpath='{.spec.template.spec.containers[?(@.name == "cluster-logging-operator")].image}')
+$ oc adm must-gather --image=quay.io/openshift/origin-cluster-logging-operator
```

Comment:
Any reason we change the image to quay.io/openshift/origin-cluster-logging-operator?